docs/community/glossary.md

```diff
@@ -0,0 +1,56 @@
+---
+id: glossary
+title: 'OpenSauced Glossary'
+sidebar_label: OpenSauced Glossary
+keywords:
+  - glossary
+---
+
+Welcome to the glossary, where we define concepts used on our platform to help you get the most out of OpenSauced.
+
+## PR Velocity
+
+PR Velocity measures the percentage and amount of days that pull requests that are merged. It is a great way to track the progress of your open-source contributions.
```

Comment:
PR Velocity tracks how quickly PRs are being merged in by identifying the average time to completion from Pull Request.